tools/clang/test/DXC/select_samplers.hlsl

```diff
@@ -0,0 +1,124 @@
+// RUN: %dxc -fcgl -HV 2018 -T ps_6_0 %s | FileCheck %s
```

Comment:
This should not be in test/DXC, test/DXC is for cmd tests on dxc.
Should be somewhere like CodeGenDXIL/operators/select.

Reply:
Woops! I'll fix it in main in a follow up